helm/resource_release.go

```diff
@@ -189,6 +189,13 @@ func resourceRelease() *schema.Resource {
 				Computed:    true,
 				Description: "Status of the release.",
 			},
+			"overrides": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The result YAML file got by merging all passed `set`, `set_string`, `set_sensitive` and `values`",
+				// Mark as sensitive to avoid printing the diff, as it might contain sensitive data
+				Sensitive: true,
```

Comment:
_Note:_ Without `Sensitive: true` we can have a detailed diff in the plan output, so we can see all exact changes of the result YAML. I like it more than just meaningless `<sensitive> => <sensitive>`  diff

However, since `set_sensitive` has been added, that could reveal such secret values. I don't see any way to avoid it, so I had to mark this attribute ultimately `Sensitive`. Let me know if I should change this.

Reply:
Without this PR, detailed diff leaks into plan anyway, correct? What about having something like `overrides_sensitive`?

Reply:
> Without this PR, detailed diff leaks into plan anyway, correct?

Yes, that's correct. But I don't see how `overrides_sensitive` would help in this case.